application/build.gradle

```diff
@@ -29,7 +29,8 @@ dependencies {
         exclude group: 'com.google.guava', module: 'guava'
     }
     runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.16'
-    runtime group: 'org.liquibase', name: 'liquibase-core', version: '3.8.9'
+    runtime group: 'org.liquibase', name: 'liquibase-core', version: '3.10.3'
```

Comment:
with this new version do we still nee the various:
`<column autoIncrement="true" name="id" type="serial">` 
we added on the previous PR?